docs/src/app/components/PropTypeDescription.jsx

```diff
@@ -51,7 +53,9 @@ const PropTypeDescription = React.createClass({
     }
 
     return (
-      <MarkdownElement text={text} />
+      <div className="propsTable">
```

Comment:
I think `propsTable` is the balance between verbosity and explicitness. :+1: for `propsTable` :grin: